confignode/src/main/java/org/apache/iotdb/confignode/service/thrift/server/ConfigNodeRPCServerProcessor.java

```diff
@@ -156,6 +162,22 @@ public DataPartitionInfo getDataPartition(GetDataPartitionReq req) throws TExcep
     return null;
   }
 
+  @Override
+  public DataPartitionInfoResp fetchDataPartitionInfo(FetchDataPartitionReq req) throws TException {
+    return null;
+  }
+
+  @Override
+  public SchemaPartitionInfoResp fetchSchemaPartitionInfo(FetchSchemaPartitionReq req)
+      throws TException {
+    return null;
```

Comment:
Will return null cause trouble? Might it be better to return a resp with empty maps?